src/GoogleStorageAdapter.php

```diff
@@ -262,13 +265,17 @@ public function read($path)
     {
         // TODO: can this be optimised to not perform 2 x api calls here?
         $object = $this->getObject($path);
-        $object = $this->normaliseObject($object);
-        $contents = $this->service->objects->get($this->bucket, $path, ['alt' => 'media']);
-        if ($contents === false) {
-            $contents = null;
+
+        try {
+            $response = $this->service->getClient()->execute(new Request('GET', $object->getMediaLink()));
+        } catch (\Exception $e) {
+            return false;
```

Comment:
Do we want to "mask" the exception here and just return false, or should we just let the exception bubble up?